source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/inputhandler/ComboBoxInputHandler.java

```diff
@@ -24,6 +24,12 @@ public String getInput()
     {
         // no need to validate
         ChoiceSetInput choiceSetInput = (ChoiceSetInput) m_baseInputElement;
-        return (String) getSpinner().getSelectedItem();
+        int index = getSpinner().getSelectedItemPosition();
+        String selectedItem = "";
+        if (index >= 0)
+        {
+            selectedItem = choiceSetInput.GetChoices().get(index).GetValue();
```

Comment:
Simply return here.. why store?

Reply:
I would prefer having one single return point in this case, after all it's just one extra line of code

Reply:
Not needed as the SDK prefers single point of return .